js/UcrBuildings.js

```diff
@@ -23,3 +23,8 @@ getBuildingLocation = function (bldg, room) {
 
     return "<a target=\"_blank\" href=\"" + url + "\">" + name + " " + room + "</a>";
 };
+
+getBuildingDesc = function(bldg) {
+  var name = buildingNames[bldg];
+  return name;
+}
```

Comment:
Makes sense, but given it's a one-liner, I'd personally inline it. We'll see what @BradleyCai thinks of it.